src/Purl/Url.php

```diff
@@ -309,9 +309,14 @@ public function getNetloc()
     public function getUrl()
     {
         $this->initialize();
-        return self::httpBuildUrl(array_map(function($value) {
+        
+        $parts = array_map(function($value) {
             return (string) $value;
-        }, $this->data));
+        }, $this->data);
+        
+        if(!$this->isAbsolute()) return self::httpBuildUrlRelative($parts);
```

Comment:
Format this like:

    if(!$this->isAbsolute()) {
        return self::httpBuildUrlRelative($parts);
    }